test/test_configuration_space.py

```diff
@@ -919,6 +919,33 @@ def test_substitute_hyperparameters_in_conditions(self):
         self.assertEqual(new_conditions[0], test_conditions[0])
         self.assertEqual(new_conditions[1], test_conditions[1])
 
+    def test_substitute_hyperparameters_in_inconditions(self):
+        cs1 = ConfigurationSpace()
+        a = UniformIntegerHyperparameter('a', lower=0, upper=10)
+        b = UniformFloatHyperparameter('b', lower=1., upper=8., log=False)
+        cs1.add_hyperparameters([a, b])
+
+        cond = InCondition(b, a, [1, 2, 3, 4])
+        cs1.add_conditions([cond])
+
+        cs2 = ConfigurationSpace()
+        sub_a = UniformIntegerHyperparameter('a', lower=0, upper=10)
+        sub_b = UniformFloatHyperparameter('b', lower=1., upper=8., log=False)
+        cs2.add_hyperparameters([sub_a, sub_b])
+        new_conditions = cs1.substitute_hyperparameters_in_conditions(cs1.get_conditions(), cs2)
+
+        test_cond = InCondition(b, a, [1, 2, 3, 4])
+        cs2.add_conditions([test_cond])
+        test_conditions = cs2.get_conditions()
+
+        self.assertEqual(new_conditions[0], test_conditions[0])
+        self.assertEqual(new_conditions[0].get_parents(), test_conditions[0].get_parents())
+        self.assertFalse(new_conditions[0].get_parents() is test_conditions[0].get_parents())
```

Comment:
```suggestion
        self.assertIs(new_conditions[0].get_parents(), test_conditions[0].get_parents())
```

Reply:
I think it should be `assertIsNot`. I changed it accordingly